bin/rebuild-support.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
we'll not call this with the default python, but instead be explicit and pass the script to the proper Python as such we don't need shebang lines.